apps/portals/src/configurations/elportal/synapseConfigs/computational_tools.ts

```diff
@@ -38,7 +38,7 @@ const computationalTools: SynapseConfig = {
     cardConfiguration: computationalCardConfiguration,
     shouldDeepLink: true,
     name: 'Computational Tools',
-    facetsToPlot: ['grant', 'softwareType'],
+    facetsToPlot: ['project'],
```

Comment:
Fix E2E test (no visualizations were being shown, since these columns do not exist).  Project makes the most sense right now given the data (since it has more than one value)